intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

```diff
@@ -98,22 +107,6 @@ After creating the results directory above, let's re-run `dir.exists()` to see i
 dir.exists("results")
```

Comment:
Can't comment but the comment above should not say "re-check" anymore.

Reply:
I think it should? The first check was on line 82 (also hidden by default in this `files changed` though!).

Reply:
@jashapiro just want to make sure you've seen this comment before I head to merge

Reply:
@jashapiro quick ping for a quick look!

Reply:
Yeah, it's fine.

Reply:
thanks, didn't want to miss something silly again like in the other dir_create PR :)